src/Tizen.Network.WiFi/Tizen.Network.WiFi/WiFiManager.cs

```diff
@@ -51,10 +51,18 @@ public override bool IsInvalid
         /// </summary>
         protected override bool ReleaseHandle()
         {
-            Interop.WiFi.Deinitialize(this.handle);
-            this.SetHandle(IntPtr.Zero);
+            if (this.handle != IntPtr.Zero)
+            {
+                Interop.WiFi.Deinitialize(this.handle);
+                this.SetHandle(IntPtr.Zero);
+            }
             return true;
         }
+
+        internal void Release()
```

Comment:
Why provide Release method? it already IDisposable object.
And it is not good to call ReleaseHandle directly.